openpype/hosts/blender/plugins/load/load_abc.py

```diff
@@ -60,16 +60,30 @@ def _process(self, libpath, asset_group, group_name):
 
         imported = lib.get_selection()
 
+        empties = [obj for obj in imported if obj.type == 'EMPTY']
+
+        container = None
+
+        for empty in empties:
+            if not empty.parent:
+                container = empty
+                break
+
+        assert container, "No asset group found"
+
         # Children must be linked before parents,
         # otherwise the hierarchy will break
         objects = []
+        nodes = list(container.children)
 
-        for obj in imported:
+        for obj in nodes:
             obj.parent = asset_group
 
-        for obj in imported:
+        bpy.data.objects.remove(container)
+
+        for obj in nodes:
             objects.append(obj)
-            imported.extend(list(obj.children))
+            nodes.extend(list(obj.children))
```

Comment:
How about this...
```suggestion
        for obj in nodes:
            nodes.extend(list(obj.children))
```

Reply:
I think I'm missing something 😅 I need to add all those objects to a list for the next step! But thinking about this, I think I can still improve it a bit, now that Blender includes `children_recursive` to get all the children, and I can avoid extending the list I'm looping :)